src/ensembl/src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors.ts

```diff
@@ -54,5 +54,9 @@ export const isEntityViewerSidebarOpen = (state: RootState): boolean => {
     : false;
 };
 
-export const getEntityViewerSidebarModalView = (state: RootState) =>
-  getEntityViewerGenomeState(state)?.sidebarModalView ?? null;
+export const getEntityViewerSidebarModalView = (state: RootState) => {
+  const activeGenomeId = getEntityViewerActiveGenomeId(state);
+  return activeGenomeId
+    ? getEntityViewerGenomeState(state)?.sidebarModalView
+    : null;
+};
```

Comment:
What was the purpose for updating this function? Why is the new form preferable?

Reply:
This shouldn't be preferable. I was checking if the getting the state with/without knowing the `genomeId` was causing any issues. So this is only a remnant of that and should be reverted.

Reply:
✅